tests/test_memory.py

```diff
@@ -1772,27 +1776,36 @@ def test_symbolic_force_access(self):
             mem.read(addr2, 5)
 
     def test_getlibc(self):
-        import manticore.utils.mappings
+        from manticore.native import mappings
         import ctypes
 
+        old_cdll = ctypes.cdll
+        old_mapping_sys = mappings.sys
+
         ctypes.cdll = mock.MagicMock()
-        manticore.utils.mappings.sys = mock.MagicMock()
+        mappings.sys = mock.MagicMock()
```

Comment:
Note that those monkey patches here are super dangerous as we don't revert them.

I added a simple reverting of them, but if an assertion fails, this won't be reverted (shall be done with some context manager).